osu.Game/Screens/Play/SquareGraph.cs

```diff
@@ -75,7 +75,7 @@ public Color4 FillColour
             }
         }
 
-        private readonly LayoutValue layout = new LayoutValue(Invalidation.DrawSize);
+        private readonly LayoutValue layout = new LayoutValue(Invalidation.DrawSize, InvalidationSource.Self);
```

Comment:
This can't be good. It means the layout won't refresh with sizing changes (such as relative size). For example if you enter skin editor before entering gameplay (or while in the player loader) and then close the skin editor, you'll get the following:

![image](https://user-images.githubusercontent.com/1329837/190565301-23876829-d295-45b2-b80d-274ae8a6c902.png)

Reply:
I see, i misunderstood how `InvalidationSource` affects invalidation, on the other hand by setting `InvalidationSource` to `InvalidationSource.Default` the `LayoutValue` will be invalidated if either one of `this.DrawSize` or `Parent.DrawSize` changed, am i correct?

Reply:
Yup, that's correct.

Reply:
Thanks for clarifying, this is fixed.